extensions/sql-migration/src/constants/strings.ts

```diff
@@ -262,6 +262,12 @@ export function STORAGE_GB(storage: number): string {
 export const RECOMMENDED_STORAGE_CONFIGURATION = localize('sql.migration.sku.targetStorageConfiguration.recommendedStorageConfiguration', "Recommended storage configuration");
 export const EPHEMERAL_TEMPDB = localize('sql.migration.sku.targetStorageConfiguration.ephemeralTempdb', "Place tempdb on the local ephemeral SSD (default D:\\) drive");
 export const LOCAL_SSD = localize('sql.migration.sku.targetStorageConfiguration.local.SSD', "Local SSD");
+export const DISKTYPE_STANDARDHDD = localize('sql.migration.sku.targetStorageConfiguration.disktype.standardHdd', "StandardHDD");
+export const DISKTYPE_STANDARDSSD = localize('sql.migration.sku.targetStorageConfiguration.disktype.StandardSsd', "StandardSSD");
+export const DISKTYPE_PREMIUMSSD = localize('sql.migration.sku.targetStorageConfiguration.disktype.PremiumSsd', "PremiumSSD");
+export const DISKTYPE_ULTRASSD = localize('sql.migration.sku.targetStorageConfiguration.disktype.UltraSsd', "UltraSSD");
+export const DISKTYPE_PREMIUMSSDV2 = localize('sql.migration.sku.targetStorageConfiguration.disktype.PremiumSsdV2', "PremiumSSDV2");
+
```

Comment:
should these SKU names be expanded from code to a readable format as:  "Standard HDD, Standard SDD, Premium SSD, etc"

Reply:
taken the change.